puppetlabs_spec_helper.gemspec

```diff
@@ -1,25 +1,86 @@
+# Generated by jeweler
+# DO NOT EDIT THIS FILE DIRECTLY
+# Instead, edit Jeweler::Tasks in Rakefile, and run 'rake gemspec'
 # -*- encoding: utf-8 -*-
-$LOAD_PATH.unshift File.expand_path("../lib", __FILE__)
-require "puppetlabs_spec_helper/version"
 
 Gem::Specification.new do |s|
-  s.name        = "puppetlabs_spec_helper"
-  s.version     = PuppetlabsSpecHelper::Version::STRING
-  s.authors     = ["Puppet Labs"]
-  s.email       = ["modules-dept@puppetlabs.com"]
-  s.homepage    = "http://github.com/puppetlabs/puppetlabs_spec_helper"
-  s.summary     = "Standard tasks and configuration for module spec tests"
+  s.name = "puppetlabs_spec_helper"
+  s.version = "1.0.1"
```

Comment:
I ended up just putting back the old Version::STRING so it would not look so weird.